### ISSUE TITLE
[BUG] 1000000000 seconds instead of 1 second?

### ISSUE THREAD

**Issue by tkafka** [superwall-me/Superwall-iOS#136]
Potential error I noticed:

`let oneSecond = TimeInterval(1_000_000_000)`

I think this should be `TimeInterval(1)`, as TimeInterval is [specified in seconds](https://developer.apple.com/documentation/foundation/timeinterval), right?

https://github.com/superwall-me/Superwall-iOS/blob/aaf3d6ed793978d3248b469c19f6f4cb948b90cb/Sources/SuperwallKit/Network/Custom%20URL%20Session/TaskRetryLogic.swift#LL23C34-L23C34

**Comment by yusuftor** [comment]
Yeah `TimeInterval` is just a typealias for double. There's no issue here although maybe it'd be better to have a Nanosecond typealias instead for clarity.
